<ui version="4.0" >
 <class>GraphinDetailFormClass</class>
 <widget class="QDialog" name="GraphinDetailFormClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>GraphinDetailForm</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" colspan="3" >
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="lbCameraDll" >
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text" >
        <string>カメラDLL</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="leCameraDLL" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="lbDeviceName" >
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text" >
        <string>デバイスの種類</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="leDeviceName" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="lbDeviceRevision" >
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text" >
        <string>デバイスのリビジョン</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="leDeviceRevision" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="lbDeviceDLL" >
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text" >
        <string>デバイスDLL</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="leDeviceDLL" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="lbDeviceDllVersion" >
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text" >
        <string>デバイスDLLのバージョン</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QLineEdit" name="leDeviceDllVersion" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="lbConfigFile" >
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text" >
        <string>コンフィグファイル</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="QLineEdit" name="leConfigFile" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0" >
      <widget class="QLabel" name="lbDeviceState" >
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text" >
        <string>デバイスの状態</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" >
      <widget class="QLineEdit" name="leDeviceState" >
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="pbOutputLog" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text" >
      <string>ログを出力</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>95</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="pbClose" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text" >
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
